mkdocs.yml

```diff
@@ -107,6 +107,12 @@ nav:
             - 'Logo customization': variants/clevo_nv41/custom_logo.md
             - 'Openness score': variants/clevo_nv41/openness_analysis.md
             - 'LVFS report': variants/clevo_nv41/lvfs_report.md
+        - 'Tuxedo IBS15':
+            - 'Releases': variants/tuxedo_ibs15/releases.md
+            - 'Building': variants/tuxedo_ibs15/building.md
```

Comment:
We should unify the section names as in the issue https://github.com/Dasharo/dasharo-issues/issues/37

Reply:
Renamed the doc to `Installation`